src/hypertrace/agent/autoinstrumentation/sitecustomize.py

```diff
@@ -0,0 +1,48 @@
+'''Enable instrumentationon all supported modules.''' # pylint: disable=R0401
+import os
+import logging
+from hypertrace.agent import Agent
+
+DEFAULTS = [
+    'flask',
+    'mysql',
+    'postgresql',
+    'grpc:server',
+    'grpc:client',
+    'requests',
+    'aiohttp'
+]
+
+# Initialize logger
+logger = logging.getLogger(__name__)  # pylint: disable=C0103
+
+MODULES = ''
+if 'HT_INSTRUMENTED_MODULES' in os.environ:
+    logger.debug("[env] Loaded HT_INSTRUMENTED_MODULES from env")
+    MODULES = os.environ['HT_INSTRUMENTED_MODULES']
+
+modules_array = MODULES.split(',')
+
+if len(modules_array) == 1 and modules_array[0] == '':
```

Comment:
Is `modules_array[0] == ''` needed? Not sure how split works in python.

Reply:
If an empty string is given, the result is an array with one element that is a blank string. To make it more intuitive, I changed the logic to check for a blank string before calling split.

$ cat test.py
str1 = "This is a test"
str_array = str1.split(' ')
print(str(str_array))
str2 = "Thisisatest"
str_array2 = str2.split(' ')
print(str(str_array2))
str3 = ""
str_array3 = str3.split(' ')
print(str(str_array3))

$ python test.py
['This', 'is', 'a', 'test']
['Thisisatest']
['']